easybuild/easyconfigs/n/ncurses/ncurses-6.0-foss-2016.04.eb

```diff
@@ -4,18 +4,17 @@ name = 'ncurses'
 version = '6.0'
 
 homepage = 'http://www.gnu.org/software/ncurses/'
-description = """The Ncurses (new curses) library is a free software emulation of curses in System V Release 4.0,
- and more. It uses Terminfo format, supports pads and color and multiple highlights and forms characters and
- function-key mapping, and has all the other SYSV-curses enhancements over BSD Curses."""
+description = """The Ncurses (new curses) library is a free software emulation
+ of curses in System V Release 4.0, and more. It uses Terminfo format, supports
+ pads and color and multiple highlights and forms characters and function-key
+ mapping, and has all the other SYSV-curses enhancements over BSD Curses."""
 
 toolchain = {'name': 'foss', 'version': '2016.04'}
-toolchainopts = {'optarch': True, 'pic': True}
+toolchainopts = {'pic': True}
 
 source_urls = [GNU_SOURCE]
 sources = [SOURCE_TAR_GZ]
 
-patches = ['ncurses-%(version)s_gcc-5.patch']
-
```

Comment:
Sorry about this, I must have completely overlooked that easyconfig in the branch and not realised it was there. I've pushed it back up with exactly the same contents as the one that already exists in develop.